people/service.go

```diff
@@ -56,15 +58,45 @@ func (s *peopleServiceImpl) isInitialised() bool {
 	return s.initialised
 }
 
+func (s *peopleServiceImpl) setInitialised(val bool) {
+	s.Lock()
+	s.initialised = val
+	s.Unlock()
+}
+
+func (s *peopleServiceImpl) isDataLoaded() bool {
+	s.RLock()
+	defer s.RUnlock()
+	return s.dataLoaded
+}
+
+func (s *peopleServiceImpl) setDataLoaded(val bool) {
+	s.Lock()
+	s.dataLoaded = val
+	s.Unlock()
+}
+
 func (s *peopleServiceImpl) Shutdown() error {
-	log.Info("Shutingdown...")
+	log.Info("Shuting down...")
+	s.Lock()
+	s.initialised = false
+	s.dataLoaded = false
+	defer func() {
+		s.Unlock()
```

Comment:
yeah, originally I had a lot of things happening in there but on last refactor I didn't refactor away the entire function... will update :)